scripts/gen_vimdoc.py

```diff
@@ -109,7 +109,10 @@
         # Section helptag.
         'helptag_fmt': lambda name: f'*api-{name.lower()}*',
         # Per-function helptag.
-        'fn_helptag_fmt': lambda fstem, name: f'*{name}()*',
+        'fn_helptag_fmt': lambda fstem, name: [
+            f'*{name}()*',
```

Comment:
Really this isn't necessary (because `:h name` would find it even if we just had the next line), but I did it this way just to make it explicit for API usage outside of lua (e.g. from C), where you don't use `vim.api.foo`.

Reply:
It's somewhat necessary for ambiguous cases.